.travis.yml

```diff
@@ -61,137 +78,181 @@ jobs:
 
   #  AFTER MERGE
   #  staging 1
-  - &lcm-integration-tests
-    name: staging1 - lcm integration tests
-    stage: after-merge
-    script: |
-      bundle exec rake -f lcm.rake docker:build
-      bundle exec rake -f lcm.rake docker:bundle
-      bundle exec rake -f lcm.rake test:docker:integration
-    env:
-    - VCR_ON=false
-    - GD_ENV=staging
-    rvm: jruby-9.1.14
+  # - &lcm-integration-tests
```

Comment:
Why is so much config commented out?

Reply:
Sorry... should put the WIP in the title